Source/Lib/Codec/EbEncHandle.c

```diff
@@ -1936,15 +1936,25 @@ void EbHevcSetParamBasedOnInput(
         sequenceControlSetPtr->maxInputPadBottom = 0;
     }
 
-    // Configure the padding
-    sequenceControlSetPtr->leftPadding  = MAX_LCU_SIZE + 4;
-    sequenceControlSetPtr->topPadding   = MAX_LCU_SIZE + 4;
-    sequenceControlSetPtr->rightPadding = MAX_LCU_SIZE + 4;
-    sequenceControlSetPtr->botPadding   = MAX_LCU_SIZE + 4;
-
     sequenceControlSetPtr->lumaWidth    = sequenceControlSetPtr->maxInputLumaWidth;
     sequenceControlSetPtr->lumaHeight   = sequenceControlSetPtr->maxInputLumaHeight;
 
+    if (!(sequenceControlSetPtr->lumaWidth % MAX_LCU_SIZE) && !(sequenceControlSetPtr->lumaHeight % MAX_LCU_SIZE))
+        sequenceControlSetPtr->lcuAligned = EB_TRUE;
+
+    // Configure the padding
+    if (sequenceControlSetPtr->lcuAligned) {
+        sequenceControlSetPtr->leftPadding  = 0;
```

Comment:
even if it's LCU aligned, we would need the extra padding for subpel. This might not show up in fast presets, rather in the better quality presets